src/clj/rems/api/applications_v2.clj

```diff
@@ -510,7 +514,8 @@
 (defn- hide-sensitive-information [application]
```

Comment:
hmm why do we have this and the `rems.api.applicaitons/hide-sensitive-information` functions?

Reply:
They are v1 and v2 versions of the API.

Reply:
Though I'm confused that how `:invitation-tokens` finds its way to the v2 API so that it would need to be removed here. AFAIK, only v1 code is setting `:invitation-tokens`.

Reply:
I'm at a loss about what should work and where. Seems like v2 is full of placeholders in `event-type-specific-application-view`?

`:invitation-tokens` is set by the code in the `dynamic.clj` but if that's not used in the future and if someone will later write the correct specific views then this line is obviously not needed.